src/frontend/tsx/Games.tsx

```diff
@@ -41,13 +41,13 @@ export function Games () {
             disabled={!hasGame}
             onClick={() => setDisplayGame(null)}>
             practice board</button>
-         ) : (
+        ) : (
           <button
             className='option'
-            disabled={hasGame}
+            disabled={practiceBoard === 'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1'}
```

Comment:
1. The default FEN already exists as a constant in `ts/constants/chess`
2. This is an incredibly inefficient way to determine if the button should be enabled or not. This is doing a string comparison every single time the state updates, which means it needs to compare every single character of the current FEN string to every single character of the default FEN string. The better way to do this would be to add a variable like `hasGame` named `practiceHasMoved` or something like that. When the practice board is reset, `practiceHasMoved` is set to `false`. If the user makes a move, `practiceHasMoved` is set to `true`. While `practiceHasMoved` is `false`, the "reset practice board" button is disabled.